src/metrics.cpp

```diff
@@ -330,6 +330,11 @@ void ThreadShowMetricsScreen()
         // Thank you text
         std::cout << _("Thank you for running a Zcash node!") << std::endl;
         std::cout << _("You're helping to strengthen the network and contributing to a social good :)") << std::endl;
+
+        // Security warning text
+        std::cout << std::endl;
+        std::cout << _("In order to ensure you are adequately protecting your privacy when using Zcash,") << std::endl;
+        std::cout << _("please see https://z.cash/support/security.html") << std::endl;
```

Comment:
https://z.cash/support/security/index.html (in case the redirect ever goes away).